crates/serpen/tests/fixtures/ast_rewriting/regular_import_aliases/main.py

```
@@ -0,0 +1,70 @@
+import collections.abc as abc_collections
+import urllib.parse as url_parser
+import xml.etree.ElementTree as xml_tree
+
+# Local module imports with aliases - these should also be processed
+import utils.helpers as helper_utils
+import utils.config as config_module
+
+# Regular imports without aliases - these should NOT be processed
+import math
+import random
+import datetime
```

Ruff (0.11.9)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)